src/scripts.c

```diff
@@ -32,41 +32,18 @@
 
 #include "swupd.h"
 
-static void update_kernel(void)
+static void update_boot(void)
 {
-	char *kernel_update_cmd = NULL;
+	char *boot_update_cmd = NULL;
 
 	if (strcmp("/", path_prefix) == 0) {
-		string_or_die(&kernel_update_cmd, "/usr/bin/kernel_updater.sh");
+		string_or_die(&boot_update_cmd, "/usr/bin/clr-boot-manager update");
 	} else {
-		string_or_die(&kernel_update_cmd, "%s/usr/bin/kernel_updater.sh --path %s", path_prefix, path_prefix);
+		string_or_die(&boot_update_cmd, "%s/usr/bin/clr-boot-manager update --path %s", path_prefix, path_prefix);
 	}
 
-	system(kernel_update_cmd);
-	free(kernel_update_cmd);
-}
-
-static void update_bootloader(void)
-{
-	char *bootloader_update_cmd = NULL;
-
-	if (strcmp("/", path_prefix) == 0) {
-		string_or_die(&bootloader_update_cmd, "/usr/bin/gummiboot_updaters.sh");
-	} else {
-		string_or_die(&bootloader_update_cmd, "%s/usr/bin/gummiboot_updaters.sh --path %s", path_prefix, path_prefix);
-	}
-
-	system(bootloader_update_cmd);
-	free(bootloader_update_cmd);
-
-	if (strcmp("/", path_prefix) == 0) {
-		string_or_die(&bootloader_update_cmd, "/usr/bin/systemdboot_updater.sh");
-	} else {
-		string_or_die(&bootloader_update_cmd, "%s/usr/bin/systemdboot_updater.sh --path %s", path_prefix, path_prefix);
-	}
-
-	system(bootloader_update_cmd);
-	free(bootloader_update_cmd);
+	system(boot_update_cmd);
```

Comment:
Yeah, it doesn't silence GCC, but at least reading through the code it becomes obvious.